### DISCUSSION TITLE
Question about <sorttitle></sorttitle> tag

### DISCUSSION THREAD

**Discussion by Estarna** [Komet/MediaElch#1395] [Q&A] [answered]
Hi there,

More a question than a request:

When editing an already existing .nfo file created by MediaElch, I added text in the "Sort Title" field in the prgram.
I saved it, and as expected the text tag was added in the .nfo.

I scrape the movie as new or edit it. When saving this time with the "Sort Title" field, the .nfo still contains the text.

Is it expected? I ask because I intend to put some specific titles in those fields and was hoping it would stay static in the .nfo file.

Thanks!

-Eric

**Comment by bugwelle** [comment, marked as answer]
So you mean that editing an already existing "sortTitle" doesn't save? :-)

If that is the case, it would be a bug.

**Reply by Estarna** [reply]
Example:
I put "My movie - Extended" in the "sort title". I save
I scraped the movie again, as new. Everything is reset except the field "sort title". (I guess the provider has no field/info for this). I save
In the end "My movie - Extended" in still in the "sort title" as well as in the .nfo file.

I'm asking if it's by design intended this way because I would put some movies related informations there and I would like to know if it will stay there in case of a re-scraping.

Thanks!

-Eric

**Reply by bugwelle** [reply]
Ah, yes. I didn't want to remove the sort title when a movie is scraped because some users may have added a custom sort title prior to scraping a movie.

**Reply by Estarna** [reply]
Thanks! I was looking for this. I needed a field which I can enter some specific movie info.

Regards,

-Eric
